Add stratify.py root install blog post

bmr-cymru opened the pull request.

mulkieran commented on content/stratify.md:
```diff
@@ -0,0 +1,169 @@
+Stratis has supported use as the root filesystem since version 2.4.0 and the steps
+required to prepare a system with Stratis as the root filesystem are conceptually
+straightforward: allocate storage to a system and then create a Stratis pool and file
+system, copy the desired installation content to the new filesystem and then set up a
```
"file" and "system" are two words here, best to join.

mulkieran commented on content/stratify.md:
````diff
@@ -0,0 +1,169 @@
+```
+
+This will install required packages in the Live environment, set up the Stratis root
+pool and file system, and runs anaconda to perform the installation. The process logs
````
^runs^run

bmr-cymru replied:
Thanks well spotted, I'll get those fixed up!

mulkieran reviewed (changes requested):
Looks good! Requesting minor fixes...

bmr-cymru commented:
I noticed a bunch of "file system" when I searched - those are all fixed up now as well as the grammar fix @mulkieran pointed out.

drckeefe commented on content/stratify.md:
````diff
@@ -0,0 +1,169 @@
+# Root password
+rootpw --iscrypted $6$ly/YwB6/b5R6Pezz$zZGzBKgkPZvYHtefBwC7gJKB0BpW7ANJyt0sfjIf46yKS2IGfcnhfFW6wiXJLpYeQXvIBlJG/W7wukX0/S9Wu/
+```
+
````
In my testing I found that the ks.cfg needs to be copied to /root prior to running stratify.py on Fedora 38 otherwise the stratify.py will fail with "Kickstart file /root/ks.cfg is missing".    There could be a step here to 
let the user know to run   cp ks.cfg /root/

bgurney-rh replied:
During my test run, after the Live CD desktop environment started, I opened a terminal and executed `sudo su -`; this results in a root shell that is in root's home directory (/root).

bmr-cymru commented:
Rebased made two minor edits:

* Clarify that files are downloaded to /root
* Switched to clear text password in ks.cfg (with steps to generate hashed password in post)
